### ISSUE TITLE
conda-forge package for installation of non-Python dependencies?

### ISSUE THREAD

**Issue by darothen** [atmos-cloud-sim-uj/PySDM#494]
Given this package's capability of running on the GPU via `ThrustRTC`, would it be possible to develop a `conda-forge` recipe to help users install all the pre-requisites for taking advantage of this hardware?

**Comment by slayoo** [comment]
Thanks @darothen!

The only non-Python requirement should be the CUDA toolkit/driver. There seem to be a cuda-forge CUDA toolkit item: https://anaconda.org/conda-forge/cudatoolkit

@tehAgitto @Kyetuur @piotrbartman do you have any experience with Conda-forge?

**Comment by slayoo** [comment]
@Golui has started exploring what needs to be done here.
First, it would be great to have ThrustRTC and CURandRTC available for conda, relevant requests reported:
- https://github.com/fynv/ThrustRTC/issues/13
- https://github.com/fynv/CURandRTC/issues/3

**Comment by slayoo** [comment]
Both ThrustRTC and CURandRTC are now available as conda packages! Thanks @fynv!

**Comment by slayoo** [comment]
OK, I've tried to read through the [ana]conda resources, the issues linked from @Golui's PR, and made a few attempts to advance us here... but this only led me to one key question: what is the added value of providing the conda-forge package? 

As pointed out by @Golui in the PR (https://github.com/atmos-cloud-sim-uj/PySDM/pull/575), the conda meta files do not allow to provide information on which channel to get packages from. In our case, this is tricky as ThrustRTC and CURandRTC are available on the ``fyplus`` channel, chempy is available on the ``bjodah`` channel, while pyevtk is at the ``conda-forge`` channel. Numba, cudatoolkit, scipy and numpy are all available from different channels...

Furthermore, we do not really want to make cudatoolkit a dependency as PySDM is usable without a GPU using the Numba backend. 

Finally, Anaconda users seem to be perfectly fine just opening the Anaconda prompt and typing ``pip install PySDM``, right?

To be constructive here, I've just added to the PySDM README information for Anaconda users pointing out to the channels where the dependencies can be found, and to the conda docs section on using ``pip`` with conda: https://github.com/atmos-cloud-sim-uj/PySDM/commit/f6493267709c60abe0ffd99d09844cb6e68c3ca7

@darothen, would be great to hear your feedback (apparently we do not have any fans of conda among PySDM developers or users...). Thanks!
